flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/WindowAggregateTest.scala

```diff
@@ -299,27 +299,6 @@ class WindowAggregateTest extends TableTestBase {
   }
 
   @Test
-  def testGroupWithFloorExpression() = {
-    val sql = "SELECT COUNT(*) FROM MyTable GROUP BY FLOOR(localTimestamp TO HOUR)"
```

Comment:
Yes, @fhueske . Let us merge this to the retraction branch.
I agree with you, with FLINK-6011, I think we should now only treat (TUMBLE, HOP, SESSION) as Windows, while FLOOR and others as function (and theirs return values should be grouping keys).